modules/code-deployment/src/main/java/org/apache/ignite/internal/deployunit/DeploymentManagerImpl.java

```diff
@@ -84,6 +84,8 @@ public class DeploymentManagerImpl implements IgniteDeployment, IgniteComponent
 
     private static final String TMP_SUFFIX = ".tmp";
 
+    private static final ChannelType DEPLOYMENT_CHANNEL = ChannelType.register((short) 1, "DeploymentUnits");
```

Comment:
It may even be public and be located in the interface, not implementation, but since no one uses it right now, except for this class, let's leave it here